plugins/woocommerce/.wp-env.json

```diff
@@ -10,7 +10,8 @@
         "ALTERNATE_WP_CRON": true
     },
     "mappings": {
-        "wp-cli.yml": "./tests/wp-cli.yml"
+        "wp-cli.yml": "./tests/wp-cli.yml",
+        "wp-content/mu-plugins/filter-setter.php": "./tests/e2e-pw/bin/filter-setter.php"
```

Comment:
Since WP Env is a development tool, I think this is fine (and the idea is to make it available to the E2E tests). However, shout out if it's concerning.